tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py

```diff
@@ -143,9 +132,7 @@ def _get_default_auxiliary_params(self) -> dict:
         default_auxiliary_params = super()._get_default_auxiliary_params()
         extra_auxiliary_params = dict(
             get_features_kwargs=dict(
-                valid_raw_types=[R_INT, R_FLOAT,
-                                 # R_CATEGORY,  # FIXME: Add R_CATEGORY features
-                                 R_OBJECT],
+                valid_raw_types=[R_INT, R_FLOAT, R_CATEGORY, R_OBJECT],
```

Comment:
last thing before merging: Can you comment out category until we resolve some of the issues with overfitting?